android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -56,6 +56,7 @@
     private DrawerLayout mDrawerLayout;
     private ActionBarDrawerToggle mDrawerToggle;
     private ArrayList<View> mMenuItems;
+    private int mButtonColor;
```

Comment:
I like it.  That also means we don't need to pass screen into `setNavUpButton`, it just needs to be able to set the correct state.